task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -267,7 +267,7 @@ spec:
 
         BUILDAH_ARGS=()
 
-        BASE_IMAGES=$(grep -i '^\s*FROM' "$dockerfile_path" | sed 's/--platform=\S*//' | awk '{print $2}')
+        BASE_IMAGES=$(grep -i '^\s*FROM' "$dockerfile_path" | sed 's/--platform=\S*//' | awk '{print $2}' | grep -v ^oci-archive:)
```

Comment:
Technically, if all the FROM lines are `FROM oci-archive:...` then grep will exit with 1 and kill the build

Reply:
Sharp eyes! Don't think that's likely, but pushed a revised version, in case this code gets cut-and-pasted somwhere else.